doc/api/sqlite.md

```diff
@@ -77,20 +77,24 @@ console.log(query.all());
 
 <!-- YAML
 added: v22.5.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/56991
```

Comment:
Just confirm, this PR url should remain the original or should I put the current one?